File: 250316/체질량지수/bmi.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int height = 0;
    int weight = 0;

    cin >> height >> weight;

    int bmi = (10000 * weight) / (height * height);

    cout << bmi;

    if(bmi >= 25) {
        cout << "\nObesity";
    }
}

File: 250316/체질량지수/bmi.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int height = sc.nextInt();
        int weight = sc.nextInt();

        int bmi = (10000 * weight) / (height * height);
        
        System.out.println(bmi);
        if(bmi >= 25) {
            System.out.println("Obesity");
        }
    }
}